src/utilities/util_verify.c

```diff
@@ -107,6 +118,6 @@ usage(void)
       "verify %s\n",
       progname, usage_prefix,
       "[-d dump_address | dump_blocks | dump_history | dump_layout | "
-      "dump_offsets=#,# | dump_pages] [-S] uri");
+      "dump_offsets=#,# | dump_pages] [-h] [-S] uri");
```

Comment:
From the looks of the usage string, "uri" is mandatory, even with "-h". Whereas "-h" doesn't take a uri.

@keithbostic raised another point in the Jira comment:
https://jira.mongodb.org/browse/WT-5354?focusedCommentId=2814237&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2814237

rather than passing "-h" and no uri, we could instead keep the verify API unchanged and verify just the portion of history store for missing keys that corresponds to the btree-id of the uri. I suggest we carry on with this change and get it merged and have that discussion as part of a separate ticket.

Reply:
Link to the created ticket :)
**https://jira.mongodb.org/browse/WT-5556**